tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java

```diff
@@ -1391,4 +1395,122 @@ public NumPathCoverageItem(
             }
         }
     }
+
+    public void testIso8601() {
+        List<String> comprehesiveElements =
+                List.of(
+                        "timeFormat",
+                        "quarters",
+                        "eras",
+                        "dayPeriods",
+                        "days",
+                        "months",
+                        "appendItems",
+                        "intervalFormatFallback");
+        // drop IDs unless they have y+M or M+d
+        List<String> raw =
+                Splitters.VBAR.splitToList(
+                        "MMMMd|MMMMW|MMMMW|yw|yw|yQQQ|yQQQQ|Gy|Gy|GyM|GyM|GyM|GyMd|GyMd|GyMd|GyMd|GyMEd|GyMEd|GyMEd|GyMEd|GyMMM|GyMMM|GyMMM|GyMMMd|GyMMMd|GyMMMd|GyMMMd|GyMMMEd|GyMMMEd|GyMMMEd|GyMMMEd|Md|Md|MEd|MEd|MMMd|MMMd|MMMEd|MMMEd|Ed|yM|yM|yMd|yMd|yMd|yMEd|yMEd|yMEd|yMMM|yMMM|yMMMd|yMMMd|yMMMd|yMMMEd|yMMMEd|yMMMEd|yMMMM|yMMMM");
```

Comment:
Could this be generated from 'those without y+M or M+d'?  seems fragile to have yet another static list.

Reply:
This is just for testing.